backend.py

```diff
@@ -825,13 +824,23 @@ def createDom0DiskFilesystems(install_type, disk, boot_partnum, primary_partnum,
             finally:
                 mount.unmount()
 
-def __mkinitrd(mounts, partition, package, kernel_version, fcoe_interfaces):
+#pylint: disable=consider-using-f-string
+def _generateBFS(mounts, primary_disk): #pylint: disable=invalid-name
+    rv, wwid, err = util.runCmd2(["chroot", mounts["root"], "/usr/lib/udev/scsi_id",
+                               "-g", primary_disk], with_stdout=True, with_stderr=True)
+    if rv != 0:
+        raise RuntimeError("Failed to whitelist %s with error: %s" % (primary_disk, err) )
+
+    # Remove ending line breaker
+    wwid = wwid.strip()
+
+    util.runCmd2(["chroot", mounts["root"], "/usr/sbin/multipath", "-a", wwid],
+                                with_stdout=True, with_stderr=True)
```

Comment:
Minor: why are you asking for stdout and stderr if you are using none of them? Did you want to check the result?

Reply:
oh, I used to use them, but then I regarding the usage. 
Good point, I will update.